tests/test_parallel_worker.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# coding: utf-8
+import os
+import sys
+import threading
+import time
+import typing
+import unittest
+from concurrent import futures
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from dss.util import parallel_worker
+from tests.infra import testmode
+
+
+class TestParallelWorker(unittest.TestCase):
+    @testmode.standalone
+    def test_sequential_complete(self, subtasks=5):
```

Comment:
what if subtask is 0?

Reply:
I don't understand your question. :(

Reply:
will everything work if `test_sequential_complete(subtasks=0)`

Reply:
add that test and i'll approve